.evergreen/install-cli.sh

```diff
@@ -27,31 +28,55 @@ if [ "${DOCKER_RUNNING:-}" == "true" ]; then
   UV_TOOL_DIR=$_root_dir/uv-tool
 fi
 
-. ./venv-utils.sh
+# If uv is not on path, try see if it is available from the Python toolchain.
+if ! command -v uv >/dev/null; then
+  _bin_path=""
+  if [ "Windows_NT" == "${OS:-}" ]; then
+    _bin_path="/cygdrive/c/Python/Current"
+  elif [ "$(uname -s)" != "Darwin" ]; then
+    _bin_path="/Library/Frameworks/Python.Framework/Versions/Current"
+  else
+    _bin_path="/opt/python/Current"
+  fi
+  if [ -d "${_bin_path}" ]; then
+    export PATH="${_bin_path}:$PATH"
+  fi
+fi
 
-if [ ! -d $SCRIPT_DIR/venv ]; then
+# If uv is still not available, we need a venv.
+if ! command -v uv >/dev/null; then
+  . ./venv-utils.sh
 
-  . ./find-python3.sh
+  if [ ! -d $SCRIPT_DIR/venv ]; then
```

Comment:
```suggestion
  if [ ! -d "$SCRIPT_DIR/venv" ]; then
```
ShellCheck SC2086: prevent globbing and word splitting.